swagger.json

```diff
@@ -321,6 +321,25 @@
       }
     }
   },
+  "/api/products/:id/request": {
+    "post": {
+      "tags": ["Request"],
+      "summary": "Create a new request",
+      "requestBody": {
+        "required": true,
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/components/schemas/productRequestBody"
+            }
+          }
+        }
+      }
+    },
+    "400": {
+      "description": "Invalid request"
+    }
```

Comment:
You need to return a more descriptive and detailed status doc here. This "400" status code should return an object with `success` and `data` keys with respect to the response message you send in the `/controllers/products.js`.